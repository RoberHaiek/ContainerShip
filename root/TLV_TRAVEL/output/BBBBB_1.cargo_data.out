CSQU0000005,L,1,1,1
CSQU0000006,L,1,1,2