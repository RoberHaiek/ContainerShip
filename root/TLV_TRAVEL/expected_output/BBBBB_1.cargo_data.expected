CSQU0000001,U,1,1,1
CSQU0000002,U,1,1,1
CSQU0000003,U,1,1,1
CSQU0000005,R,-1,-1,-1
CSQU0000006,R,-1,-1,-1